templates/PAAStack.yaml

```diff
@@ -63,10 +63,161 @@ Resources:
         Ref: vendorId
     UpdateReplacePolicy: Delete
     DeletionPolicy: Delete
+  mattercrlpaabucket48E65E17:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: AES256
+      BucketName: matter-crl-paa-bucket
+      PublicAccessBlockConfiguration:
```

Comment:
Looking at yamls I just remembered that we wanted to tag all the resources with `MatterStack.matterPKITag`. Let's add for consistency